app/javascript/apps/staff/components/dashboard.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <form id="userForm" onsubmit="submitForm(event)">
+    <h3>Создание клиента</h3>
+    <div>
+      <label for="fullname">Full Name:</label>
+      <input type="text" id="fullname" name="fullname" required minlength="5">
+      <div id="fullnameError" class="error"></div>
+    </div>
+    <div>
+      <label for="phone">Phone:</label>
+      <input type="tel" id="phone" name="phone" required pattern="[0-9]+">
+      <div id="phoneError" class="error"></div>
+    </div>
+    <div>
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" required>
+      <div id="emailError" class="error"></div>
+    </div>
+    <button type="submit">Submit</button>
+  </form>
+</template>
+
+<script>
+function submitForm(event) {
+  event.preventDefault();
+  const fullname = document.getElementById('fullname').value;
```

Comment:
pls set this variables like a state in data() of the component:
```
data() {
    return {
      fullname: ""
    };
  },
```
and update it through v-model in HTML input